packages/core/src/utils/zod.test.ts

```diff
@@ -57,10 +117,74 @@ describe('zodTypeToSwagger', () => {
     expect(zodTypeToSwagger(string().nullable())).toEqual({ type: 'string', nullable: true });
   });
 
+  describe('literal type', () => {
+    it('boolean', () => {
+      expect(zodTypeToSwagger(literal(true))).toEqual({
+        type: 'boolean',
+        format: 'true',
+      });
+      expect(zodTypeToSwagger(literal(false))).toEqual({
+        type: 'boolean',
+        format: 'false',
+      });
+    });
+
+    it('number', () => {
+      expect(zodTypeToSwagger(literal(-1.25))).toEqual({
+        type: 'number',
+        format: '-1.25',
+      });
+      expect(zodTypeToSwagger(literal(999))).toEqual({
+        type: 'number',
+        format: '999',
+      });
+
+      const bigIntLiteral = literal(BigInt(1_000_000_000));
+      expect(() => zodTypeToSwagger(bigIntLiteral)).toMatchError(
+        new RequestError('swagger.invalid_zod_type', bigIntLiteral)
+      );
```

Comment:
I'm not sure, but shouldn't we treat it as regular number? e.g.
```
type: 'number',
format: '1,000,000,000'
```

Reply:
According to the simeng's comment above, we'd better not support bigint literal
https://github.com/logto-io/logto/pull/1126#discussion_r898631143